docs/Consuming.md

```diff
@@ -1,10 +1,44 @@
 # Consuming messages
 
-Consuming topics can be done by right clicking a topic in the explorer or from the command palette.  Some things to note about consuming:
+Consuming topics can be done:
+
+* with [Kafka Explorer](#kafka-explorer) by right clicking a topic in the explorer.

```

Comment:
`from the [Kafka Explorer](#kafka-explorer), by right-clicking on a topic and selecting "Start Consumer".`